src/System.IO.Pipelines/tests/Configurations.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
+      netstandard;
```

Comment:
This won't work because you are directly referencing the implementation assembly, which has a higher version than the netstandard configuration. I would keep netcoreapp config, and add `netfx` and `uap` so that we also get tested there.